templates/vue/cypress/integration/lightbox/activity.spec.js

```diff
@@ -1,25 +1,88 @@
 describe("Activity", () => {
-  it("should be able to complete an activity with an audio-based answer", () => {
+  it("should be able to switch between questions in a activity", () => {
     cy.fixture("one-node.json").as("oneNode")
     cy.setup("@oneNode")
 
     cy.getSelectedNode().then(node => {
       cy.openModal("edit", node.id)
       cy.changeMediaType("activity")
-
       const question = `What's your name?`
-
+      const placeholder = "placeholder"
+      const answer = "Tapestry"
       cy.contains(/add question/i).click()
 
       cy.contains(/question text/i).click()
       cy.focused().type(question)
-      cy.contains(/audio/i).click()
+      cy.getByTestId("question-answer-text-0").click({ force: true })
+      cy.getByTestId("question-answer-text-single-0").click({ force: true })
+      cy.getByTestId("question-answer-text-single-placeholder-0").type(placeholder)
+
+      cy.contains(/add question/i).click()
+      const question2 = `What's your favorite pet?`
+      const placeholder2 = "placeholder 2"
+      const answer2 = "Dog"
+      cy.getByTestId("question-text-1").click()
+      cy.focused().type(question2)
+      cy.getByTestId("question-answer-text-1").click({ force: true })
+      cy.getByTestId("question-answer-text-single-1").click({ force: true })
+      cy.getByTestId("question-answer-text-single-placeholder-1").type(placeholder2)
 
       cy.submitModal()
       cy.openLightbox(node.id)
+      cy.route("POST", "**/quiz*").as("submit")
+      cy.lightbox().within(() => {
+        cy.get(`[placeholder="${placeholder}"]`).should("be.visible")
+        cy.get("input").type(answer)
+        cy.contains(/submit/i).click()
+        cy.contains("Thanks!").should("be.visible")
+        cy.getByTestId("completion-next-button").click()
+        cy.get("input").type(answer2)
+        cy.contains(/submit/i).click()
+        cy.contains("Thanks!").should("be.visible")
+        cy.contains(/done/i).click()
+      })
+      cy.lightbox().should("not.exist")
+    })
+  })
 
-      cy.contains(/microphone access/i, { timeout: 10000 }).should("not.exist")
+  it("should be able to edit questions they have already answered by enabling a new answerType", () => {
```

Comment:
I'm not sure I understand this test..

Reply:
@wynnset After a user enables a answerType(text type first) and answers it, the user can then edit that finished question again by enabling one more answerType(in this question, enables audio) and answers it with audio

Reply:
I see. This is a very rare occasion because the activities are not generally completed by the same user that creates the activity. Probably a better test is to make sure a different user's answers does not show up.